src/routes/webhook/webhook-context.ts

```diff
@@ -0,0 +1,28 @@
+import Logger from "bunyan";
+
+export class WebhookContext {
+	id: string;
+	name: string;
+	payload: any;
```

Comment:
I want a way to make the payload typable.  Maybe add a generic to the context that's defaulted to any if you don't specify it and that type is set as the payload type?